packages/core/src/types.ts

```diff
@@ -1553,9 +1564,9 @@ export default interface ICCSSProps {
     us?: TCSSSimplePropValue
     styleText?: (v: string) => string
     mid?: TCSSSimplePropValue
-    hScroll?: TCSSSimplePropValue
+    hScroll?: (i?: string, o?: TNestedInput) => string
```

Comment:
We don't have these anymore on core, all these functionalities are in separate packages.

Reply:
removing

Reply:
And that has no hScoll nor, only scroll, which is `scroll: boolean | 'x' | 'y'`